chainerx_cc/chainerx/routines/type_util.h

```diff
@@ -93,4 +93,15 @@ Dtype ResultType(Container args) {
     return resolver.Resolve();
 }
 
+namespace internal {
+
+inline Dtype GetMathResultDtype(Dtype dtype) {
```

Comment:
A cosmetic but how about defining this function inside the `namespace internal { ... }` that already exists at the top?

Reply:
Fixed.